packages/core-web/src/styles/markbind.css

```diff
@@ -395,6 +378,11 @@ button.copy-btn {
     width: 1em;
 }
 
+/* Fixes cases where the navbar does not fill up the entire width of viewport when changing pages */
```

Comment:
Hmm, there was a point in time during the period of experimenting with SSR where I noticed that the navbar was not filling up the width of the viewport, which was why I added this to "fix" it. 

However, I can't seem to replicate the same behaviour that I observed during that period. I think it may have been some side-effects caused by my other changes while experimenting. 

This CSS fix is unnecessary, thanks for spotting this! I will remove it accordingly :-)